client/card-readers/list/list-item.tsx

```diff
@@ -4,10 +4,19 @@
  */
 import classNames from 'classnames';
 import { __ } from '@wordpress/i18n';
+import React from 'react';
 
 /*eslint-disable camelcase*/
-const CardReaderListItem = ( { reader: { id, device_type, is_active } } ) => {
-	const status = is_active
+const CardReaderListItem: React.FunctionComponent< {
+	reader: {
+		id: string;
+		device_type: string;
+		is_active: boolean;
+	};
+} > = ( {
+	reader: { id, device_type: deviceType, is_active: isActive },
+} ): JSX.Element => {
```

Comment:
@Automattic/pulsar, @reykjalin what do you think about this approach? Should we follow it in our TS endeavor?

Reply:
I think it's a good suggestion! It might even make sense to just make a type for this:

```ts
type CardReader = {
  id: string;
  device_type: string;
  is_active: boolean;
};

function CardReaderListItem( { reader }: { reader: CardReader } ): JSX.Element {
  const { id, device_type: deviceType, is_active: isActive } = reader;
```

Reply:
I agree with @sirbrillig and @reykjalin's thoughts here.